observability/monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -228,12 +244,17 @@ endif::openshift-dedicated,openshift-rosa[]
 
 // Configuring labels for outgoing metrics
 include::modules/monitoring-adding-cluster-id-labels-to-metrics.adoc[leveloffset=+1]
-include::modules/monitoring-creating-cluster-id-labels-for-metrics.adoc[leveloffset=+2]
+
+// Creating cluster ID labels for metrics
+// The following module should only include core platform monitoring (CPM tags)
+include::modules/monitoring-creating-cluster-id-labels-for-metrics.adoc[leveloffset=+2,tags=**;CPM;!UWM]
+// The following module should only include monitoring for user-defined projects (UWM tags)
+include::modules/monitoring-creating-cluster-id-labels-for-metrics.adoc[leveloffset=+2,tags=**;!CPM;UWM]
 
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#configuring-remote-write-storage_configuring-the-monitoring-stack[Configuring remote write storage]
+* xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#configuring-remote-write-storage-cpm_configuring-the-monitoring-stack[Configuring remote write storage]
```

Comment:
Is it possible to have a link to each of the Configuring remote write storage modules (CPM and UMM)

Reply:
Hello, yes, I aggree. I was not really looking at links right now, just making sure the builds work. I have a separate Jira created that will be dedicated to preparing links. But I will add this as a note in that Jira to make sure I will not forget to do this :slightly_smiling_face: thank you!
(Here is the Jira for reference: https://issues.redhat.com/browse/OBSDOCS-1474)